src/crypto/rsa_provider/rsa_provider_impl.cpp

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <openssl/err.h>
```

Comment:
wrong order of includes: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes